server/events/markdown_renderer.go

```diff
@@ -243,7 +264,18 @@ func (m *MarkdownRenderer) renderProjectResults(results []command.ProjectResult,
 				resultData.Rendered = m.renderTemplateTrimSpace(templates.Lookup("stateRmSuccessUnwrapped"), result.StateRmSuccess)
 			}
 		} else {
```

Comment:
Could the next if block be rewritten like this?

```suggestion
		} else if !(result.Error != nil || result.Failure != "") {
```

Reply:
👍